localisation/000_Other_title_names.csv

```diff
@@ -355,4 +355,5 @@ c_dazaralor_adj;Zuldazar;Zuldazar;Zuldazar;;Zuldazar;;;;;;;;;x
 d_zuldazar;Kings Bay;;;;;;;;;;;;;x
 d_zuldazar_adj;Kings Bay;;;;;;;;;;;;;x
 d_defias_brotherhood;Defias Brotherhood;;;;;;;;;;;;;x
-d_defias_brotherhood_adj;Defias;;;;;;;;;;;;;x
\ No newline at end of file
+d_defias_brotherhood_adj;Defias;;;;;;;;;;;;;xd_hammerfall;Hammerfall;;;;;;;;;;;;;x
```

Comment:
Merge went wrong?

Reply:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1063/commits/1ca8e6e7fb9cf4fd7f2333b996d9274cd1013de5.